test/SourceKit/CodeComplete/complete_filter.swift

```diff
@@ -52,14 +52,14 @@ func foo() {
 
 // RUN: %sourcekitd-test -req=complete.open -pos=11:5 \
 // RUN:   -req-opts=filtertext=d %s -- %s > %t.d
-// RUN: %FileCheck -check-prefix=CHECKD %s < %t.d
```

Comment:
This is `D` as in `%t.d`, no need to rename

Reply:
Ah, sorry, thanks! I've reverted. (I think that is what you meant, right?)

Reply:
Yep! Thanks.